processor/src/lib.rs

```diff
@@ -70,6 +71,11 @@ pub struct StackTrace {
     aux_builder: stack::AuxTraceBuilder,
 }
 
+pub struct RangeCheckTraceTable {
+    lookups_8bit: [usize; RANGE_CHECK_TRACE_TABLE_WIDTH],
+    len: usize,
+}
+
```

Comment:
Similar comment here - I would keep this in the `range` module since it's not actually needed at this level.

Reply:
Done!